src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -43,6 +43,7 @@ namespace NUnit.VisualStudio.TestAdapter
     [FileExtension(".dll")]
     [FileExtension(".exe")]
     [DefaultExecutorUri(NUnit3TestExecutor.ExecutorUri)]
+    [Category("managed")]
```

Comment:
This line is causing `dotnet test --framework=net46` to crash VSTest.Console.exe with:

> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified.

The net46 tests use the net35 adapter, and the net35 adapter is here referencing `System, Version=2.0.0.0`.
I'm not sure why `dotnet test --framework=net46` crashes, while a direct call to VSTest.Console.exe in my Visual Studio installation does not crash. Perhaps the difference is simply in which version of VSTest is running?

/cc @OsirisTerje

Reply:
If we start building with .NET Core SDK 2.0.0, this problem disappears.

2.0.0 is the oldest SDK which solves the problem. With SDK 2.0.0, `dotnet test` runs VSTest version `15.7.0-preview-20180221-13`
1.1.9 is the newest SDK which does not solve the problem. With SDK 1.1.9, `dotnet test` runs VSTest version `15.0.0.0`.

@OsirisTerje This might be a reason to go ahead with https://github.com/nunit/nunit3-vs-adapter/pull/508?

Reply:
@jnm2 Since you have been in here, can you check this now ,  #508 has been merged.

Reply:
Yep, that fixed it. I double-checked by rebasing this branch on master on my machine.